### ISSUE TITLE
Hover and Click only work over Part of a Glued Span

### ISSUE THREAD

**Issue by JohnWSaundersIII** [zeroclipboard/zeroclipboard#330]
I'll get you more detail later, but I only _just now_ reproduced this, and wanted to get you the information I have.

I have seen this for a while, that copying doesn't always work over the entire width of a span. In particular, for copying an email address, I've seen it work at the beginning of the address, but not always at the end. I was never able to tie this down.

I happen to have had the HTML tab of Firebug open while I was clicking in my page. As I moved the mouse cursor over the span, I noticed that you add the `zeroclipboard-is-hover` class to the span. 

**I also noticed that when the cursor moved over the "@" sign of the email address, the `zeroclipboard-is-hover` class was removed.**

This may be a clue to someone who knows the code.

---

More clues. I turned on logging of mouse events for that span, started the mouse pointer before the span, and slowly moved the mouse from left to right. Here's what I saw:

> mouseover clientX=100, clientY=133 » span#ctl00__ctl01_emailLabel.MCD_A_Control_HasClick
> mousemove clientX=100, clientY=133 » span#ctl00__ctl01_emailLabel.MCD_A_Control_HasClick
> mouseout clientX=100, clientY=133 » span#ctl00__ctl01_emailLabel.MCD_A_Control_HasClick
> mouseover clientX=181, clientY=132 » span#ctl00__ctl01_emailLabel.MCD_A_Control_HasClick
> mouseout clientX=181, clientY=132 » span#ctl00__ctl01_emailLabel.MCD_A_Control_HasClick

In other words, there was one mouse{over,move,out} when the pointer first moved over the span. There was another mouse{over,out} when the pointer reached the first character after the "@". Note, no new `mousemove`.

This was with Firefox 26.0 running on Windows 7 Ultimate, using zeroclipboard 1.2.3.
